pytket/phir/phirgen_parallel.py

```diff
@@ -30,23 +30,24 @@
 logger = logging.getLogger(__name__)
 
 
-def exec_order_preserved_helper(
-    ordered_dict: OrderedDict["UnitID", int], group_number: int, qubit_last_group: int
+def exec_order_preserved(
+    group_exec_order: list[int], group_number: int, qubit_last_group: int
 ) -> bool:
-    """A helper to determine whether order is preserved when adding qubits to groups."""
-    # determine whether the current group number is later in execution
-    # than the last group in which a qubit was used
+    """A helper to determine whether current group number a qubit is eligible
+    or is later in execution than the last group in which a qubit was used.
+    """  # noqa: D205
```

Comment:
```suggestion
    """Determine whether order is preserved when adding qubits to groups."""
```
Let's keep it simple as you have a similar comment on lines 46--47 below.